release_notes/368-release-notes.adoc

```diff
@@ -53,14 +55,20 @@ Release date: February 14, 2022
 This issue has been fixed.
 Central now reports an error for such cases.
 
+[id="resolved-in-version-3682"]
+=== Resolved in version 3.68.2
```

Comment:
I see that we sometimes add newer versions above older and sometimes below (see for example how 3.68.1, line 73 is higher up than 3.68.1, line 80). We have similar inconsistency for 3.67 release notes. It would be great if we settle on one scheme — I don't have a preference which one.

Reply:
Good idea - I'll fix this and any others that I'm working on. I think going forward I'll put recent changes first to call them out. Thanks for bringing this up!